### PR TITLE
Add receptorctl manpages

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -97,8 +90,43 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     [author], 1)
+    (master_doc, "receptor", "Receptor Documentation", [author], 1),
+    ("getting_started_guide/index", "receptor_getting_started_guide", "Receptor Getting Started Guide", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]: I don't think we need to build the getting started guide, or other guides, as a manpageq, do we?

File: docs/source/conf.py [hunk 2]
```diff
@@ -97,8 +90,43 @@
...
+    ("porting_guide/index", "receptor_porting_guide", "Receptor Porting Guide", [author], 1),
+    ("receptorctl/receptorctl_index", "receptorctl", "receptor client", [author], 1),
+    (
+        "receptorctl/receptorctl_connect",
+        "receptorctl-connect",
+        "receptorctl connect establishes a connection between local client and a Receptor node",
+        [author],
+        1,
+    ),
+    ("receptorctl/receptorctl_ping", "receptorctl-ping", "receptorctl ping tests the network reachability of Receptor nodes", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_ping", "receptorctl-ping", "Tests the network reachability of Receptor nodes", [author], 1),
```

File: docs/source/conf.py [hunk 3]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_reload", "receptorctl-reload", "receptorctl reload reloads the Receptor configuration for the connected node", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_reload", "receptorctl-reload", "Reloads the Receptor configuration for the connected node", [author], 1),
```

File: docs/source/conf.py [hunk 4]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_status", "receptorctl-status", "receptorctl status displays the status of the Receptor network", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_status", "receptorctl-status", "displays the status of the Receptor network", [author], 1),
```

File: docs/source/conf.py [hunk 5]
```diff
@@ -97,8 +90,43 @@
...
+    ),
+    (
+        "receptorctl/receptorctl_version",
+        "receptorctl-version",
+        "receptorctl version displays version information for receptorctl and\
+        the Receptor node to which it is connected.",
+        [author],
+        1,
+    ),
+    ("receptorctl/receptorctl_work_cancel", "receptorctl-work-cancel", "receptorctl work cancel terminates one or more units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_cancel", "receptorctl-work-cancel", "Terminates one or more units of work", [author], 1),
```

File: docs/source/conf.py [hunk 6]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_work_list", "receptorctl-work-list", "receptorctl work list displays known units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_list", "receptorctl-work-list", "Displays known units of work", [author], 1),
```

File: docs/source/conf.py [hunk 7]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_work_release", "receptorctl-work-release", "receptorctl work release deletes one or more units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_release", "receptorctl-work-release", "Deletes one or more units of work", [author], 1),
```

File: docs/source/conf.py [hunk 8]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_work_results", "receptorctl-work-results", "receptorctl work results gets results for units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_results", "receptorctl-work-results", "Gets results for units of work", [author], 1),
```

File: docs/source/conf.py [hunk 9]
```diff
@@ -97,8 +90,43 @@
...
+    ("receptorctl/receptorctl_work_submit", "receptorctl-work-submit", "receptorctl work submit requests a Receptor node to run a unit of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_submit", "receptorctl-work-submit", "Requests a Receptor node to run a unit of work", [author], 1),
```

File: docs/source/conf.py [hunk 10]
```diff
@@ -97,8 +90,43 @@
...
+        [author],
+        1,
+    ),
+    ("receptorctl/receptorctl_ping", "receptorctl-ping", "receptorctl ping tests the network reachability of Receptor nodes", [author], 1),
+    ("receptorctl/receptorctl_reload", "receptorctl-reload", "receptorctl reload reloads the Receptor configuration for the connected node", [author], 1),
+    ("receptorctl/receptorctl_status", "receptorctl-status", "receptorctl status displays the status of the Receptor network", [author], 1),
+    (
+        "receptorctl/receptorctl_traceroute",
+        "receptorctl-traceroute",
+        "receptorctl traceroute Displays the network route that packets follow to Receptor nodes",
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
        "Displays the network route that packets follow to Receptor nodes",
```

File: docs/source/conf.py [hunk 11]
```diff
@@ -97,8 +90,43 @@
...
+        [author],
+        1,
+    ),
+    (
+        "receptorctl/receptorctl_version",
+        "receptorctl-version",
+        "receptorctl version displays version information for receptorctl and\
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
        "Displays version information for receptorctl and\
```

File: docs/source/conf.py [hunk 12]
```diff
@@ -97,8 +90,37 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     [author], 1)
+    ("receptorctl/receptorctl_index", "receptorctl", "receptor client", [author], 1),
+    (
+        "receptorctl/receptorctl_connect",
+        "receptorctl-connect",
+        "Establishes a connection between local client and a Receptor node",
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
        "Establishes a connection between local client and a Receptor node.",
```

File: docs/source/conf.py [hunk 13]
```diff
@@ -97,8 +90,37 @@
...
+        [author],
+        1,
+    ),
+    ("receptorctl/receptorctl_ping", "receptorctl-ping", "Tests the network reachability of Receptor nodes", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_ping", "receptorctl-ping", "Tests the network reachability of Receptor nodes.", [author], 1),
```

File: docs/source/conf.py [hunk 14]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_reload", "receptorctl-reload", "Reloads the Receptor configuration for the connected node", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_reload", "receptorctl-reload", "Reloads the Receptor configuration for the connected node.", [author], 1),
```

File: docs/source/conf.py [hunk 15]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_status", "receptorctl-status", "Displays the status of the Receptor network", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_status", "receptorctl-status", "Displays the status of the Receptor network.", [author], 1),
```

File: docs/source/conf.py [hunk 16]
```diff
@@ -97,8 +90,37 @@
...
+    ),
+    (
+        "receptorctl/receptorctl_version",
+        "receptorctl-version",
+        "Displays version information for receptorctl and\
+        the Receptor node to which it is connected.",
+        [author],
+        1,
+    ),
+    ("receptorctl/receptorctl_work_cancel", "receptorctl-work-cancel", "Terminates one or more units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_cancel", "receptorctl-work-cancel", "Terminates one or more units of work.", [author], 1),
```

File: docs/source/conf.py [hunk 17]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_work_list", "receptorctl-work-list", "Displays known units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_list", "receptorctl-work-list", "Displays known units of work.", [author], 1),
```

File: docs/source/conf.py [hunk 18]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_work_release", "receptorctl-work-release", "Deletes one or more units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_release", "receptorctl-work-release", "Deletes one or more units of work.", [author], 1),
```

File: docs/source/conf.py [hunk 19]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_work_results", "receptorctl-work-results", "Gets results for units of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_results", "receptorctl-work-results", "Gets results for units of work.", [author], 1),
```

File: docs/source/conf.py [hunk 20]
```diff
@@ -97,8 +90,37 @@
...
+    ("receptorctl/receptorctl_work_submit", "receptorctl-work-submit", "Requests a Receptor node to run a unit of work", [author], 1),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    ("receptorctl/receptorctl_work_submit", "receptorctl-work-submit", "Requests a Receptor node to run a unit of work.", [author], 1),
```

**Review by oraNod — APPROVED** [review]
LGTM. Some minor nits for consistency with punctuation.

@thom-at-redhat I also notice the formatting is inconsistent. Is this coming from black / ruff? Just wondering why some lists are broken into multiple lines and others are not.

**Comment by thom-at-redhat** [comment]
> LGTM. Some minor nits for consistency with punctuation.
> 
> @thom-at-redhat I also notice the formatting is inconsistent. Is this coming from black / ruff? Just wondering why some lists are broken into multiple lines and others are not.

Yes, it's black that reformats it :-/

**Review by resoluteCoder — APPROVED** [review]

**Pull request merged**